src/components/Movie/Movie.js

```diff
@@ -1,10 +1,16 @@
 import React from "react";
 import './Movie.css';
+import Hero from "../Hero/Hero";
+import Genre from "../Genre/Genre";
+import Media from "../Media/Media";
+import MovieDescription from "../MovieDescription/MovieDescription";
 
 const Movie = ({data, clearSelectedMovie}) => {
     return (
-        <section>
-            <h2>{data.title}</h2>
+        <section className='Movie'>
+            <Hero info={data}/>
+            <MovieDescription info={data} />
+            {data.genres.map(genre => <Genre key={Date.now()} genre={genre}/>)}
```

Comment:
At first I thought maybe using Date.now() in this context might cause React to unneccessarily re-render these components a lot because each time the map runs a new Date will be created for the keys but that won't matter anyways because this page will get completely re-rendered every time it's visited anyways!

Reply:
okay cool. i wasn't really sure what to put as the key and thats what i came up with on the fly

Reply:
Was kinda cool to see how React generated those Genre components so fast that Date.now() ended up having duplicate keys. Using the genre as the key was a smart fix!